### DISCUSSION TITLE
Svg selection with transparent area

### DISCUSSION THREAD

**Discussion by roonie007** [simonwep/viselect#155] [Q&A] [answered]
How can I select this SVG element only when the mouse is over the line

![image](https://user-images.githubusercontent.com/7860859/150621866-97242a9d-a048-4b52-9e53-d5881d71c7e0.png)

What is actually happening, is when ever the mouse enter the SVG RectBound element even on the transparent area the item is selected (check the picture below):

![image](https://user-images.githubusercontent.com/7860859/150621914-4b29fefe-8ba0-4b24-8fb7-8c79a4087d38.png)

What I am looking for is, how to trigger the selection only when the mouse is over the line ?
Thanks

**Comment by simonwep** [comment, marked as answer]
Hi! This won't be possible without major changes to how this library works internally... like it would not only consider an _area_ of elements to select but also some who were "touched" via `mouseenter`... I'll consider your issue a feature request and see what I can do :)
